libs/sdk-ui-filters/src/AttributeFilter@next/AttributeDropdown/AttributeDropdownList.tsx

```diff
@@ -49,37 +52,51 @@ export const AttributeDropdownList: React.FC<IAttributeDropdownListProps> = ({
     searchString,
     isMobile,
 }) => {
+    const intl = useIntl();
+
     const getItemKey = useCallback((i: AttributeListItem) => {
-        const isSelectionByUri = !!selectedItems[0]?.uri;
-        return isNonEmptyListItem(i) ? (isSelectionByUri ? i.uri : i.title) : "empty";
+        return isNonEmptyListItem(i) ? (i.uri ? i.uri : i.title) : "empty";
```

Comment:
Is this change correct? (fact that some item has uri is not equal to the check that selection has uri?)

Reply:
This method return key of item ... so when ATT mount than selection is empty (negative filter) so the result was that key was always title (What about two attributes with same title) I was afraid that fact that key changed when selection changes could be problem. I do not understand correctly why in previous impl was this check ... and I will always sand title rather then this check .... 

But true is when selection changed then next render could it fix. 

So should I change this impl ???? To previous one ?